{{ cookiecutter.project_slug }}/frontend/Makefile.default

```diff
@@ -52,13 +52,13 @@ format-lint: ## Format Code with Lint
 .PHONY: format
 format: format-prettier format-stylelint format-lint ## Format the codebase according to our standards
 
-.PHONY: i18n
-i18n: ## Sync i18n
-	yarn i18n
+.PHONY: preinstall
+preinstall:
+	if [ -f $$(pwd)/mrs.developer.json ]; then if [ -f $$(pwd)/node_modules/.bin/missdev ]; then yarn develop; else yarn develop:npx; fi; fi
```

Comment:
No more yarn develop or yarn develop:npx:

https://6.dev-docs.plone.org/volto/upgrade-guide/index.html#upgrade-to-use-yarn-3

```diff
+.PHONY: install
+install: ## Install the frontend
+       @echo "Install frontend"
+       $(MAKE) omelette
+       $(MAKE) preinstall
+       yarn install
+
+.PHONY: preinstall
+preinstall: ## Preinstall task, checks if missdev (mrs-developer) is present and runs it
+       if [ -f $$(pwd)/mrs.developer.json ]; then make develop; fi

+
+.PHONY: develop
+develop: ## Runs missdev in the local project (mrs.developer.json should be present)
+       npx -p mrs-developer missdev --config=jsconfig.json --output=addons --fetch-https
+
+.PHONY: omelette
+omelette: ## Creates the omelette folder that contains a link to the installed version of Volto (a softlink pointing to node_modules/@plone/volto)
+       if [ ! -d omelette ]; then ln -sf node_modules/@plone/volto omelette; fi
+
+.PHONY: patches
+patches:
+       /bin/bash patches/patchit.sh > /dev/null 2>&1 ||true
```

need to be there.